packages/core/components/Puck/index.tsx

```diff
@@ -477,7 +477,6 @@ export function Puck({
                       <div
                         style={{
                           border: "1px solid var(--puck-color-grey-8)",
-                          transform: "scale(1, 1)",
```

Comment:
Assume this will get wiped out as already merged

Reply:
Yup - will rebase!